### DISCUSSION TITLE
Meeting Agenda 3/26

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#350] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning
3. Meet with Darian

Note: Victor is out sick

**Comment by moreno-michael** [comment]
Meeting minutes 3/26

1. Task Review
- Issue #1295 completed
- Issue #1450 in progress
- Issue #1385 in progress
2. Sprint Planning
- Issue #1427 assigned to Michael
- Issue #1294 reassigned to Victor
3. Meet with Darian
- Asked about ethernet port and outlet access during expo, Kunal will follow up with Carson
- Hard deadline for poster submissions and project files is April 10th, posters will be printed for us if we submit by this date
- Sprint tasks discussed
- Sentry and logrocket implementation discussed
